lib/tiktok/getVideoComments.js

```
@@ -0,0 +1,34 @@
+import getActorStatus from "../apify/getActorStatus";
+import getDataset from "../apify/getDataset.js";
+import getFormattedCommentsInfo from "./getFormattedCommentsInfo.js";
+import getVideoCommentsDatasetId from "./getVideoCommentsDatasetId.js";
+
+const getVideoComments = async (postURLs, chat_id) => {
+  try {
+    const datasetId = await getVideoCommentsDatasetId(postURLs);
+    let attempts = 0;
+    const maxAttempts = 30;
+    let progress = 0;
+    while (1) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)